src/environments/environment.ts

```diff
@@ -6,6 +6,8 @@
 export const environment = {
   production: false,
   apiEndpoint: 'https://zonemaster.net/api',
+  //apiEndpoint: 'https://zonemaster.ppsfleet.navy/api',
```

Comment:
What is it? What is it for? How and when is it used?

Reply:
Sorry, I pushed too much stuff, will revert.